frontend/test/pytest/test_callback.py

```diff
@@ -395,5 +395,46 @@ def f(x):
     assert np.allclose(np.sin(1.0 / 2.0), f(1.0 / 2.0))
 
 
+@pytest.mark.parametrize(
+    "arg",
+    [0.1, jnp.array(0.1), jnp.array([0.1]), jnp.array([0.1, 0.2]), jnp.array([[1, 2], [3, 4]])],
+)
+def test_accelerate_device(arg):
+    """Test with device parameter"""
+
+    @accelerate(dev=jax.devices()[0])
+    def identity(x):
+        return x
+
+    @qml.qjit
+    def qjitted_fn(x):
+        return identity(x)
+
+    assert np.allclose(qjitted_fn(arg), arg)
```

Comment:
@erick-xanadu are you able to test that `identity` is running on the expected device?

Reply:
are you also able to test that `identity` is being jitted?

Reply:
Yep! I mean more so, have an assertion in the test suite, so that this is continued to be tested as part of CI :)

Reply:
I am not sure we can programmatically do this :thinking:

Reply:
@dime10 I don't think this can be done.